localstack-core/localstack/services/events/rule.py

```diff
@@ -29,6 +28,21 @@
 
 
 class RuleService:
+    rule: Rule
```

Comment:
Can these be `None`? Typing doesn't match with the constructor

Reply:
fixed